evm/src/NttManager/NttManager.sol

```diff
@@ -393,7 +393,9 @@ contract NttManager is INttManager, NttManagerState {
         // construct the NttManagerMessage payload
         bytes memory encodedNttManagerPayload = TransceiverStructs.encodeNttManagerMessage(
             TransceiverStructs.NttManagerMessage(
-                seq, toWormholeFormat(sender), TransceiverStructs.encodeNativeTokenTransfer(ntt)
+                bytes32(uint256(seq)),
```

Comment:
Any particular reason why you decided to cast to uint256 here rather then changing the counter to a uint256? No strong feelings either way personally

Reply:
I suppose it's in its own storage slot anyway. would it be crazy to consider packing the sequence counter into the same slot as the outbound rate limit params? they are always accessed together, so would be a nontrivial gas saving.

Reply:
Nice suggestion actually, that could be a follow up PR. Should be a pretty low risk change as gas optimisations go